### ISSUE TITLE
There's no way to know the parent dimension for a collection of values

### ISSUE THREAD

**Issue by cdeszaq** [yahoo/fili#285]
In the values collection resource, we should have a link back to it's parent dimension so that it's easy to navigate and makes the association clear and explicit.

Something like `"dimension": "http://example.com/v1/dimensions/theDimension"`

**Comment by michael-mclawhorn** [comment]
Isn't this a breadcrumb issue?  If a bunch of dimensions share a domain load, how will you know which link to show?
Do we need to carry the request api path down in order to be able to add the path-back to the collection?


Michael Mclawhorn

Senior Software Development Engineer

mclawhor@yahoo-inc.com

2021 S First St Suite 110, Champaign, IL, 61820, US

Phone (408) 349-3300  Fax (408) 349-3301


On Thursday, May 18, 2017, 9:51:54 AM CDT, Rick Jensen <notifications@github.com> wrote:
In the values collection resource, we should have a link back to it's parent dimension so that it's easy to navigate and makes the association clear and explicit.

Something like "dimension": "http://example.com/v1/dimensions/theDimension"

**Comment by cdeszaq** [comment]
Yes, this definitely has some interesting interactions if / when we split domain and dimension into their own concepts. If we go that route, what is currently the Dimension resource is instead the Domain resource (for purposes of this issue), and Dimension isn't really a consideration there.

That said, if we expose values for a Dimension (ie. a subset of the values of the backing Domain), we would probably want links in that collection to both the backing domain, as well as some indicator of the path (ie. the Dimension).
